cmd/satellite/open_stream.go

```diff
@@ -15,12 +15,13 @@
 package satellite
 
 import (
+	"errors"
 	"fmt"
+	"github.com/spf13/cobra"
 	"log"
 	"os"
 	"os/signal"
-
-	"github.com/spf13/cobra"
```

Comment:
I think this previous grouping was correct. I'm not 100% sure of golang import best practices, but I think core imports are usually grouped together, then external imports can be grouped together.